bench-tps/src/bench.rs

```diff
@@ -233,9 +233,13 @@ pub fn do_tx_transfers(
     leader: &NodeInfo,
     shared_tx_thread_count: &Arc<AtomicIsize>,
     total_tx_sent_count: &Arc<AtomicUsize>,
+    thread_batch_sleep_ms: usize,
 ) {
     let client = mk_client(&leader);
     loop {
+        if thread_batch_sleep_ms > 0 {
+            sleep(Duration::new(0, thread_batch_sleep_ms as u32 * 1_000_000));
```

Comment:
`Duration::from_ms(thread_batch_sleep_ms)` should be a bit cleaner here.

Reply:
@sakridge fixed, thank you! (from a rust newbie)